app/views/partials/_nav_bar.html.erb

```diff
@@ -0,0 +1,29 @@
+<nav role="navigation" class="nav-extended">
+  <div class="nav-content">
+    <div class="container">
+      <%= image_tag "logos/CMU_Logo_Horiz_White.png", id: "cmu-logo" %>
+    </div>
+  </div>
+
+  <div class="nav-wrapper container">
+    <a id="logo-container" href="http://www.cmu.edu/" class="brand-logo left">

```

Comment:
Should we have the PDS logo go to the PDS site and the Carnegie Mellon University logo go to the CMU main page?